src/veins/modules/mobility/traci/ParBuffer.h

```diff
@@ -0,0 +1,89 @@
+#pragma once
```

Comment:
As a new file, shouldn't we add the standard "this is part of Veins" comment block at the top here?